### DISCUSSION TITLE
How to get rid off the welcome screen?

### DISCUSSION THREAD

**Discussion by GlaedrOromis** [inducer/pudb#665] [General]
![image](https://github.com/user-attachments/assets/9a3d1fbd-f9ed-416b-a7e0-724d42d139e5)
I'm using VSC and I can't press OK, I can't enter any keys, I can't exit the screen. I need to close the whole terminal because there's no way to interact with the screen.

**Comment by inducer** [comment]
Could you talk about the use case a bit? pudb is intended as an interactive debugger, and so without any way to interact with it (IIUC?), the experience won't be great, and I'm not sure that's fixable without writing an essentially entirely different piece of software.
